pypeit/spectrographs/spectrograph.py

```diff
@@ -821,6 +855,11 @@ def wavegrid(self, binning=None, midpoint=False,samp_fact=1.0):
         return np.power(10.0,loglam_grid)
 
 
+    @property
+    def telluric_grid_file(self):
+        """Return the grid of HITRAN atmosphere models for telluric correctinos"""
+        pass
```

Comment:
Should this instead return None?  How is this handled in the code that calls this method?

Reply:
This property has been removed because we put it in the parset.